src/altinn_model_publisher/service/altinn_mongo_service.py

```diff
@@ -0,0 +1,44 @@
+"""Service layer module for saving altinn models to a zip file."""
+import gzip
+import os
+
+from datacatalogtordf.catalog import Catalog
+from pymongo import MongoClient
+
+
+ALTINN_MODELS_MONGO_ID = "altinn-models"
+MONGO_COLLECTION = "models"
+MONGO_DATABASE = "altinnModelPublisher"
+MONGO_CONNECTION_STRING = f"""mongodb://{os.environ['MONGO_USERNAME']}:{os.environ['MONGO_PASSWORD']}@mongodb:27017"""
+
+
+def save_catalog_to_mongo(catalog: Catalog) -> None:
+    """Zip and save catalog to mongo."""
+    connection = MongoClient(MONGO_CONNECTION_STRING)
```

Comment:
Hadde initialisert client utenfor denne funksjonen for å unngå å lage en ny connection per request. Alternativt, tror det er best å lukke client med `connection.close()` for å unngå for mange åpne connections med mindre MongoClient er en singleton

Reply:
ee14a44